give.php

```diff
@@ -356,6 +356,8 @@ private function setup_constants() {
 		private function includes() {
 			global $give_options;
 
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
```

Comment:
@ravinderk Yes, we can do this as well.. I'll implemented this change. Also, i've added support for Yoast SEO Premium.